test/e2e/trigger_dependency_annotation_test.go

```diff
@@ -59,6 +59,9 @@ func TestTriggerDependencyAnnotation(t *testing.T) {
 	pod := resources.EventLoggerPod(subscriberName)
 	client.CreatePodOrFail(pod, lib.WithService(subscriberName))
 
+	// Wait for subscriber to become ready
+	client.WaitForAllTestResourcesReadyOrFail()
```

Comment:
I think this is fine, I had replaced L88 with this line for my local testing.

Reply:
And it worked?